### PR TITLE
Record vaccinations: display actions and outcomes

### PR DESCRIPTION
**Pull request by benilovj**
## What is this change?

Update the designs of the record vaccinations index page to show actions and outcomes.

## Before

image

## After

image

### CODE REVIEW

File: app/views/vaccinations/_patient_row.html.erb [hunk 1]
```diff
@@ -6,12 +6,8 @@
...
   </td>
-  <td class="govuk-table__cell">
-    <%= patient.consent %>
-  </td>
-  <td class="govuk-table__cell" data-testid="child-status">
-    <%= render(AppStatusTagComponent.new(status: PatientSession.human_enum_name("outcome", outcome),
-                                         colour: vaccination_status_colour(outcome),
-                                         icon: vaccination_status_icon(outcome))) %>
+  <td class="govuk-table__cell" data-testid="child-action">
+    <% label = PatientSession.human_enum_name("actions_or_outcome", action_or_outcome[:action] || action_or_outcome[:outcome]) %>
```
[the remark this answers is not in the document]
Comment (benilovj) [reply app/views/vaccinations/_patient_row.html.erb:10]: yes, that's right.

[the remark this answers is not in the document]
Comment (misaka) [reply app/views/vaccinations/_patient_row.html.erb:10]: Surprising, but also convenient. Does this mean it shouldn't have `enum` in the name? 🤔

File: app/controllers/vaccinations_controller.rb [hunk 2]
```diff
@@ -90,13 +91,26 @@ def set_patient
     @patient = Patient.find(params[:id])
   end
 
-  def set_patient_outcomes
-    @patient_outcomes =
+  def set_patient_consents_triages_and_vaccs_records
```
Comment (misaka) [inline app/controllers/vaccinations_controller.rb:94]: Might be worth naming this after the variable it sets:

```suggestion
  def set_patient_details
```

**Review by misaka — APPROVED** [review]
🚀

**Pull request merged**